src/main/fc/settings.yaml

```diff
@@ -693,6 +693,11 @@ groups:
         min: 0
         max: 100
         default_value: 1.0
+      - name: airspeed_address
```

Comment:
let's rename it to `i2c_address_ms4525`

Reply:
You can't do that without increasing the size of the compressed name character set, which will cost around 500b on every target.,

Reply:
* The character set is shared across all settings names
* It is currently 5 bit (32 characters)
* We use 31 with NUL, a-z, 1-3
* You can't have both 4 & 5

Reply:
lovely

Reply:
trivial to go to a 6 bit set if we're prepared to endure the size increase

Reply:
I don't know what new name to put, because "ms4525" won't be possible

Reply:
Convert 4525 to hex ms0x11AD

Reply:
> Convert 4525 to hex 0x11AD?


no, the lay user cannot read in hexadecimal.

Reply:
> Convert 4525 to hex 0x11AD?

In any case, the compression character set only contains lower case letters ...